src/org/jitsi/impl/neomedia/MediaStreamStatsImpl.java

```diff
@@ -1209,13 +1209,12 @@ private void setRttMs(long rttMs)
             {
                 // RemoteBitrateEstimator is a CallStatsObserver and
                 // VideoMediaStream has a RemoteBitrateEstimator.
-                MediaStream mediaStream = this.mediaStreamImpl;
+                MediaStreamImpl mediaStream = this.mediaStreamImpl;
```

Comment:
To be able to do `mediaStream.getRemoteBitrateEstimator()` (I removed the getter from the interface).